include/mvd/bits/mvd3_misc.hpp

```diff
@@ -206,6 +210,21 @@ inline std::vector<std::string> MVD3File::getSynapseClass(const Range & range) c
 }
 
 
+inline std::vector<TSV::TSVInfo> MVD3File::getTSVInfo(const Range & range ) const{
+    std::vector<std::string> etypes = getEtypes(range);
+    std::vector<boost::int32_t> layers = getLayers(range);
+    std::vector<std::string> morphologies = getMorphologies(range);
+    std::vector<TSV::TSVInfo> TSVInfos;
+    for (auto mecombo_tupple : boost::combine(etypes, layers, morphologies)) {
```

Comment:
Combine is really neat! :) But it seems we are doing two copies. (to the tuple, then individual vars)
The good old way would be to have three const iterators, or even a plain indexed 'for loop', which won't suffer for this problem (actually there would be no copy at all).
It's already better if we don't extract the fields to individual vars

Reply:
If combine creates copy, maybe `boost::zip_iterator` is a better fit?

Reply:
There's heaptrack to check :)

Reply:
Excellent!